lexibank_wold.py

```diff
@@ -24,41 +32,83 @@ class WOLDLexeme(Lexeme):
     original_script = attr.ib(default=None)
 
 
-class Dataset(clld.CLLD):
-    __cldf_url__ = (
-        "http://cdstar.shh.mpg.de/bitstreams/EAEA0-92F4-126F-089F-0/wold_dataset.cldf.zip"
-    )
+def normalize_text(text):
+    text = text.replace("\n", " // ")
+    text = re.sub("\s+", " ", text).strip()
+
+    return text
+
+
+class Dataset(BaseDataset):
+    __cldf_url__ = "http://cdstar.shh.mpg.de/bitstreams/EAEA0-92F4-126F-089F-0/wold_dataset.cldf.zip"
     dir = Path(__file__).parent
     id = "wold"
     lexeme_class = WOLDLexeme
 
-    def cmd_install(self, **kw):
-        ccode = {
-            x.attributes["wold_id"]: x.concepticon_id for x in self.conceptlist.concepts.values()
-        }
+    def cmd_download(self, args):
+        if not self.raw_dir.exists():
+            self.raw_dir.mkdir()
+
+        files = ["borrowings.csv"]
+        self.raw_dir.download_and_unpack(
+            self.__cldf_url__, *[Path(f) for f in files]
+        )
 
-        fields = self.lexeme_class.fieldnames()
-        with self.cldf as ds:
-            vocab_ids = [v["ID"] for v in self.original_cldf["contributions.csv"]]
+    def cmd_makecldf(self, args):
+        # add the bibliographic sources
+        args.writer.add_sources()
 
-            self.add_sources(ds)
+        # add the languages from the language file
+        # NOTE: the source lists all languages, including proto-languages,
+        # but the `forms` only include the first 41 in the list
+        languages = self.raw_dir.read_csv("languages.csv", dicts=True)
+        for language in languages:
+            args.writer.add_language(
+                ID=language["ID"],
```

Comment:
Do you NEED to set the ID as an ID here? I'd prefer to have an ID that reminds of the language name. It would be easier to check data afterwards.